src/ui/components/VerifyPasscode/VerifyPasscode.tsx

```diff
@@ -113,6 +113,7 @@ const VerifyPasscode = ({
       onDidDismiss={() => handleClearState()}
     >
       <ResponsivePageLayout
+        activeStatus={isOpen}
```

Comment:
How did we get to this change? Was there a bug here before?

Reply:
Yes. I've an error after click connect button in choose identifier stage, verify passcode render empty page. It seem `useIonViewDidEnter` not running. But now because we aren't using this action in connect new wallet connection, I will revert it.